static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -621,6 +623,7 @@ define([
     function shouldRenderLabel($adSlot) {
         return !$adSlot.hasClass('ad-slot--frame') &&
```

Comment:
Yes there is, but this is a bigger change that will have ramifications in other creatives. If you want take a stab at it 😃